include/os/linux/kernel/linux/simd_x86.h

```diff
@@ -322,7 +441,7 @@ kfpu_end(void)
 #define	kfpu_init()		0
 #define	kfpu_fini()		((void) 0)
 
-#endif /* defined(HAVE_KERNEL_FPU_INTERNAL) */
+#endif /* defined HAVE_KERNEL_FPU_INTERNAL || HAVE_KERNEL_FPU_XSAVE INTERNAL */
```

Comment:
space in `HAVE_KERNEL_FPU_XSAVE INTERNAL` (also a better phrasing would be `defined(HAVE_KERNEL_FPU_INTERNAL) || defined(HAVE_KERNEL_FPU_XSAVE_INTERNAL)` or `defined(HAVE_KERNEL_FPU_INTERNAL || HAVE_KERNEL_FPU_XSAVE_INTERNAL)`)

Reply:
Thanks for catching that, fixed.